lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -1,6 +1,6 @@
 # Copyright 2021 Canonical Ltd.
 # See LICENSE file for licensing details.
-"""## Overview.
+"""## Overview
```

Comment:
The easy way out of 
```
D415 First line should end with a period, question mark, or exclamation point
```
is to have the period in there.
Alternative suggestions welcome!

Reply:
Oh yeah how did I not see it; I think ideally that line should be a one-line description of the library, though it feels a little repetitive with the overview right below it.

I'm applying this suggestion in the meantime, seeing if anyone has better ideas!